case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+---
+title: Azure DevOps Case Study
+keywords: development platforms, Azure DevOps, Microsoft
+tags: Development Platforms, Azure DevOps, Microsoft
+sidebar: development-platforms
+permalink: development-platforms/case-study-azure-devops.html
+folder: development-platforms
+---
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [Brief History](#brief-history)
+3. [Features](#features)
+4. [Strengths](#strengths)
+5. [Weaknesses](#weaknesses)
+6. [Pricing](#pricing)
+7. [Comparison](#comparison)
+8. [How To](#how-to)
+9. [Summary](#summary)
+10. [Credits](#credits)
+11. [References](#references)
+
+## **Introduction**
+
+<div style="width: 600px;">
```

Comment:
```suggestion
```

The text is rendered inside a container so this is only needed for previewing

Reply:
Will fix it. 
Added that for my preview, for me it just didn't look good in the preview